client/src/components/AboutUs/AboutUsContainer.js

```diff
@@ -70,6 +91,134 @@ export const MissionAndVisionBannerContainer = styled.div`
   }
 `;
 
+export const HeadingContainer = styled.div`
+  width: 881px;
+  height: 200.6px;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  margin: 0 auto;
+  h1{
+    font-family: Poppins;
+    font-size: 38px;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.23;
+    letter-spacing: normal;
+    color: var(--color-primary-text);
+    margin-top:48.6px;
+  }
+  p{
+    font-family: Poppins;
+    font-size: 22px;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.36;
+    letter-spacing: normal;
+    color: var(--color-primary-text);
+  }`;
+
+export const OurStoryContainer = styled.div`
+h1 {
+width: 1031px;
+height: 66px;
+font-family: Poppins;
+align-items: center;
+justify-content: center;
+text-align: center;
+margin: 0 auto;
+marginLeft: 124px;
+marginRight: 126px;
+font-size: 44px;
+font-weight: bold;
+font-stretch: normal;
+font-style: normal;
+line-height: normal;
+letter-spacing: normal;
+color: var(--color-primary-text);
+}
+h2 {
+  align-items: center;
+justify-content: center;
+text-align: center;
+margin: 0 auto;
+padding-top: 45.2px;
+padding-bottom: 13.2px;
+marginLeft: 675.2px;
+marginRight: 3382.2px;
+  width: 12.4x;
+  height: 16.6px;
+  background-color: var(--color-primary-brand-color);
+  color: #425af2;
+}
```

Comment:
Use `theme` colors, fonts, etc, use `rem`s not `px`, use

Reply:
Same for all of these components. below and above

Reply:
same as previous comment

Reply:
done